DOCS/canonical-identity-platform-tutorial-deploy-identity-bundle.md

```diff
@@ -1,47 +1,27 @@
-# Introduction
+The Canonical Identity Platform is a composable identity broker based on open source products from [Ory](https://www.ory.sh/open-source/), [Postgres](https://www.postgresql.org/) and [Traefik Labs](https://traefik.io/).
```

Comment:
Always spell the product name as in the upstream. E.g., here, it should be `PostgreSQL` (even if common usage tolerates `Postgres`).